docs/content/concepts/assets/asset-observations.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Asset Observations | Dagster
+description: Dagster provides functionality to record metadata about assets.
+---
+
+# Asset Observations
+
+An asset observation is an event that records metadata about a given asset.
+
+## Relevant APIs
+
+| Name                                                    | Description                                                          |
+| ------------------------------------------------------- | -------------------------------------------------------------------- |
+| <PyObject module="dagster" object="AssetObservation" /> | Dagster event indicating that an asset's metadata has been recorded. |
+| <PyObject module="dagster" object="AssetKey" />         | A unique identifier for a particular external asset                  |
+
+## Overview
+
+<PyObject module="dagster" object="AssetObservation" /> events are used to communicate
```

Comment:
I think the first sentence might be clearer as "are used to record metadata in Dagster about a given asset".